custom/icds_reports/reports/prevalence_of_undernutrition.py

```diff
@@ -328,7 +333,10 @@ def get_prevalence_of_undernutrition_sector_data(domain, config, loc_level, loca
         if sql_location.name not in result_set:
             chart_data['blue'].append([sql_location.name, 0])
 
-    chart_data['blue'] = sorted(chart_data['blue'])
+    chart_data['blue'] = sorted(
+        chart_data['blue'],
+        key=lambda loc_and_value: (loc_and_value[0] is not None, loc_and_value)
```

Comment:
@calellowitz This feels like an error to me for sql_location to not have a name. Any ideas if this is just in test data and won't be hit in prod?

Reply:
does this even happen in test data? but yeah agree that locations should always have names. I would have guessed that was enforced at the db level, although it appears not to be. How does this change affect the sorting?

Reply:
Python 3 does not permit sorting iterables containing strings or numbers when a value of ```None``` is present.